integration_tests/macros/plugins/snowflake/prep_external.sql

```diff
@@ -1,18 +1,22 @@
 {% macro snowflake__prep_external() %}
 
     {% set external_stage = target.schema ~ '.dbt_external_tables_testing' %}
+    {% set parquet_file_format = target.schema ~ '_dbt_external_tables_testing_parquet' %}
```

Comment:
@b-per I think this should be a dot?

```sql
target_schema_dbt_external_tables_testing_parquet
```

instead of:

```sql
target_schema.dbt_external_tables_testing_parquet
```

Reply:
Sure, I have updated it